src/content/docs/logs/forward-logs/google-cloud-platform-log-forwarding.mdx

```diff
@@ -7,373 +7,450 @@ tags:
+  >
+
+  Create a bucket in GCP to hold the generated Dataflow template by running the following command:
+
+  ```bash
+  gsutil mb -p ${PROJECT_ID} -l ${REGION} gs://${BUCKET_NAME}
+  ```
+
+  </Collapser>
+
+  <Collapser
+    id="dataflow-account"
+    title="5. Create a service account."
+  >
+
+  Execute the following commands:
+
+  1. Create the service account:
+
+    ```bash
+    gcloud iam service-accounts create ${SERVICE_ACCOUNT_NAME}
+    ```
+
+  2. Grant permissions to the service account:
+    
+    ```bash
+    gcloud projects add-iam-policy-binding ${PROJECT_ID} --member="serviceAccount:${SERVICE_ACCOUNT_NAME}@${PROJECT_ID}.iam.gserviceaccount.com" --role="roles/owner"
+    ```
+  
+  3. Generate the service account key file:
+
+    ```bash
+    gcloud iam service-accounts keys create ${SERVICE_ACCOUNT_KEY_FILENAME} --iam-account=${SERVICE_ACCOUNT_NAME}@${PROJECT_ID}.iam.gserviceaccount.com
+    ```
+
+  4. Reference your service account key file using the `GOOGLE_APPLICATION_CREDENTIALS` environment variable, as it will be used by subsequent commands:
+
+    ```bash
+    export GOOGLE_APPLICATION_CREDENTIALS=${SERVICE_ACCOUNT_KEY_FILENAME}
+    ```
+  
+  </Collapser>
+
+  <Collapser
+    id="dataflow-compile"
+    title="6. Compile and publish the PubsubToNewRelic template."
+  >
+
+  Execute the following command:
+
+  ```bash
+  mvn compile exec:java \
+      -Dexec.mainClass=com.google.cloud.teleport.templates.PubsubToNewRelic \
+      -Dexec.cleanupDaemonThreads=false \
+      -Dexec.args=" \
+          --project=${PROJECT_ID} \
+          --region=${REGION} \
+          --enableStreamingEngine \
+          --stagingLocation=gs://${BUCKET_NAME}/staging/ \
+          --gcpTempLocation=gs://${BUCKET_NAME}/temp/ \
+          --templateLocation=gs://${BUCKET_NAME}/template/PubsubToNewRelic \
+          --runner=DataflowRunner \
+      "
+  ```
+
+  </Collapser>
+
+  <Collapser
+    id="dataflow-job"
+    title="7. Run the template as a Dataflow job."
+  >
+
+  Execute the following command to start shipping logs using a Dataflow job that reads from your Pub/Sub topic:
+
+
+  ```bash
+  gcloud dataflow jobs run ${JOB_NAME} \
+      --gcs-location=gs://${BUCKET_NAME}/template/PubsubToNewRelic \
+      --region=${REGION} \
+      --parameters "inputSubscription=projects/${PROJECT_ID}/subscriptions/${INPUT_SUBSCRIPTION_NAME},licenseKey=${NR_LICENSE_KEY}"
+  ```
+
+  This command only requires these two values:
+  
+  * The input PubSub subscription used to read log messages
+  * The New Relic license key used to send your logs
+  
+  For other values, the command uses default configuration settings that you can further [customize as needed](#dataflow-config).
+
+  </Collapser>
+
+  <Collapser
+    id="dataflow-config"
+    title="8. Optional: Tune up your Dataflow log forwarder job."
+  >
+
+  Here is a reference of available options you can use to further tune up the execution of your Dataflow log forwarder job.
+
+  <table>
```

Comment:
Consider expanding the column width for that first column in the following table - some of the parameter names are pretty smushed, and the descriptions will more easily wrap.